### PR TITLE
Linting webapi and fixing a few problems

### PR DESCRIPTION
**Pull request by miigotu**

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

File: sickbeard/webapi.py [hunk 1]
```diff
@@ -273,6 +275,8 @@ class ApiCall(ApiHandler):
 
     def __init__(self, args, kwargs):
         # missing
+        super(ApiCall, self).__init__(*args, **kwargs)
+
```
Comment (MGaetan89) [inline sickbeard/webapi.py]: Did you try this?
I wanted to added it a while back, but it introduced some issues. Can remember which one though.

**Comment by miigotu** [comment]
@MGaetan89 all of this is untested so far so I just pushed it up to be able to see my changes in a cleaner diff than in a terminal =P

I started this and got sidetracked around 4am.

**Comment by miigotu** [comment]
Still a few strange try/excepts that seem to do nothing here, but this is much better than it was.

See lines 363, 293, 296, 298

**Pull request merged**